### ISSUE TITLE
Ability to close system message with keyboard

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#1110]
When a system message appears from a page load (not an ajax system message/ ex: 'Welcome to GCconnex %s' on login) users cannot close the message with their keyboard.
